File: Vepr-bot/models/order.py
class Order:

	def __init__(self):
		self.user = 0
		self.comment = " "
		self.id = 0
		self.total_price = 0

	@property
	def _total_price(self):
		return self.total_price

	@_total_price.setter
	def _total_price(self, value):
		self.total_price = value


	@property
	def _user(self):
		return self.user

	@_user.setter
	def _user(self, value):
		self.user = value



	@property
	def _comment(self):
		return self.comment

	@_comment.setter
	def _comment(self, value):
		self.comment = value



	@property
	def _id(self):
		return self.id

	@_id.setter
	def _id(self, value):
		self.id = value


	def get(self):
		orderDict = self.__dict__
		del orderDict['id']
		orderDict['idUser'] = orderDict.pop('user')
		return orderDict

	def set(self, user, comment, id, total_price):
		self.user = user
		self.comment = comment
		self.id = id
		self.total_price = total_price
		return self

File: vepr_backend/lib/model/order.dart
import 'package:conduit/conduit.dart';
import 'package:vepr_backend/model/order_contents.dart';
import 'package:vepr_backend/model/order_status.dart';
import 'package:vepr_backend/model/user.dart';


class Order extends ManagedObject<_Order> implements _Order {}
class _Order {
  @primaryKey
  int? id;
  @Column(indexed: true, useSnakeCaseName: true, nullable: true)
  String? comment;
  @Column(indexed: true, useSnakeCaseName: true, nullable: false)
  double? totalPrice;
  @Column(indexed: true, useSnakeCaseName: true, nullable: false, defaultValue: "now()")
  DateTime? orderDate;

  @Serialize(input: true, output: false)
  int? idUser;
  
  ManagedSet<OrderStatus>? orderStatusList;

  ManagedSet<OrderContents>? orderContentsList;

  @Relate(#orderList, isRequired: true, onDelete: DeleteRule.cascade)
  User? user;
}